components/Dashboard/Modals/ProfileDialog/AccountDetails.js

```diff
@@ -106,6 +147,11 @@ const AccountDetails = ({ session, editMode, saveChanges, setSaveChanges }) => {
             variant="outlined"
             fullWidth
             value={username}
+            error={username !== '' && (!validUsername || serverError)}
+            helperText={
+              !validUsername &&
+              'Must be between 8-12 alphanumeric, underscore, and dot characters. Underscore and dot cannot be adjacent.'
```

Comment:
lol why is this. esp the "Underscore and dot cannot be adjacent" part

Reply:
So users don't have ______ as a username